src/parameters/Parameters.jl

```diff
@@ -4,17 +4,46 @@ import ClimaCore as CC
 import Insolation.Parameters as IP
 import Thermodynamics as TD
 import CloudMicrophysics as CM
+import SurfaceFluxes as SF
 
 abstract type AbstractClimaAtmosParameters end
 const ACAP = AbstractClimaAtmosParameters
 
+abstract type AbstractTurbulenceConvectionParameters end
+const ATCP = AbstractTurbulenceConvectionParameters
+
 Base.broadcastable(param_set::ACAP) = tuple(param_set)
+Base.broadcastable(param_set::ATCP) = tuple(param_set)
+
+Base.@kwdef struct TurbulenceConvectionParameters{FT} <: ATCP
```

Comment:
Not sure if we want to keep edmf parameters in a separate struct or have them be part of ClimaAtmos parameters below. Right now things are mixed up a little. I can clean it up in a separate PR, once we decide @szy21

Reply:
At least right now we don't put `microphysics` parameters inside `turbconv` parameters so that we can create `thermodynamics` parameters...

Reply:
I think it would be good to have edmf parameters in a separate struct. We can do it after this.

Reply:
Actually, I think I'll do it before. Stay tuned ;)